api-server/src/roster/views.py

```diff
@@ -28,15 +28,24 @@
 
 class AviatorListView(ListCreateAPIView):
 
+    serializer_class = AviatorSerializer
+    permission_classes = [permissions.IsAuthenticatedOrReadOnly]
     queryset = Aviator.objects.all().order_by('-rank_code', 'position_code')
+
+
+class AviatorFromUserListView(RetrieveUpdateDestroyAPIView):
+
     serializer_class = AviatorSerializer
     permission_classes = [permissions.IsAuthenticatedOrReadOnly]
 
+    def get_queryset(self):
+        return Aviator.objects.filter(user__id=self.kwargs["pk"])
```

Comment:
```suggestion
        return Aviator.objects.get(user__id=self.kwargs["pk"])
```

we're only going to match one